stories/Input/CheckboxStory.tsx

```diff
@@ -22,6 +22,14 @@ const props = {
       require: 'yes',
       description: 'Sets the label of checkbox.',
     },
+    {
+      name: 'small',
```

Comment:
do you think using `size: "large" | "small" ` will be more scalable?

Reply:
Ooh if I use size and keep default one as small, then I won't have to change  both the projects. Thanks!